stylesheets/_session_theme.scss

```diff
@@ -52,26 +68,17 @@
 
       a {
         text-decoration: underline;
-        color: var(--color-received-message-text);
+        color: inherit;
```

Comment:
Do we need this? Shouldn't it just inherit from the parent. Unless we override all a tags somewhere

Reply:
we need it actually